.github/workflows/ubuntu.yml

```diff
@@ -61,7 +61,7 @@ jobs:
             wget https://github.com/WizardMac/ReadStat/releases/download/v1.1.9/readstat-1.1.9.tar.gz
             tar -xzf readstat-*.tar.gz && cd readstat-*/
             ./configure
-            make
+            make CFLAGS='-Wno-error' CXXFLAGS='-Wno-error'
```

Comment:
Uhm, I turn this on on purpose usually?
Shouldnt we just do: `-Wno-error=use-after-free` instead?

Reply:
Turning it on will make the build fail, we only do this in the workflow and not any local builds. 
Fully fixing it would require upstream code changes (I remember you fixed this for upstream once, but not sure why it's happening again)

Reply:
Maybe some compiler got updated to be more strict or something, but this fixes it so looks good ^^